app/views/offices/_form.html.slim

```diff
@@ -27,6 +27,6 @@
             .columns.small-12.end
               = b.label do
                 = b.check_box
-                = b.text
+                = "#{b.text} (#{@becs.where(jurisdiction_id: b.value).first.code})"
```

Comment:
I think we should avoid doing sql queries in the views at any cost. So ideally the `@becs` would either be pre-processed or the objects in the `@jurisdictions` array would somehow be decorated and include the `bec` straight away (not sure exactly how).